library/include/borealis/list.hpp

```diff
@@ -123,6 +123,7 @@ class SelectListItem : public ListItem
     SelectListItem(std::string label, std::vector<std::string> values, unsigned selectedValue = 0, std::string description = "");
 
     void setSelectedValue(unsigned value);
+    int getSelectedValue();
```

Comment:
Can you replace that by `unsigned` please?

Reply:
done